beetsplug/musicbrainz.py

```diff
@@ -901,7 +791,7 @@ def album_for_id(
             self._log.debug("Invalid MBID ({}).", album_id)
             return None
 
-        res = self.api.get_release(albumid)
+        res = self.mb_api.get_release(albumid, includes=RELEASE_INCLUDES)
```

Comment:
I was a bit thrown of by this `includes=`, from the API docs and was expecting this to be `inc=`. I see that it is properly parsed in `get_entity`. This parameter needs a bit of documentation imo.